tools/srs_tools/val_metadata.xml

```diff
@@ -0,0 +1,67 @@
+<tool id="srs_metadata" name="Validate ISO 19139" version="@VERSION@" profile="20.01">
+    <description>metadata documents</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="4.2.1">r-base</requirement>
+        <requirement type="package" version="2.12.0">r-r.utils</requirement>
+        <requirement type="package" version="3.5_21">r-raster</requirement>
+        <requirement type="package" version="1.0_7">r-sf</requirement>
+        <requirement type="package" version="1.5_32">r-rgdal</requirement>
+        <requirement type="package" version="0.5_9">r-rgeos</requirement>
+        <requirement type="package" version="0.5_5">r-stars</requirement>
+        <requirement type="package" version="1.4.1">r-stringr</requirement>
+        <requirement type="package" version="1.19">r-ncdf4</requirement>
+        <requirement type="package" version="0.7">r-geometa</requirement>
+        <requirement type="package" version="1.4.3">r-httr</requirement>
+        <requirement type="package" version="3.99_0.9">r-xml</requirement>
+        <requirement type="package" version="1.3.3">r-xml2</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+            '$__tool_directory__/val_metadata.r'
+            '$input'
+            '$output' 1> Metadata_validation.txt
```

Comment:
@Marie59 is it correct that you only want to use the output from stdout and NOT what is written to $output?

You are picking with `from_work_dir="Metadata_validation.txt"` so I think $output is ignored.